ocs_ci/utility/deployment_openshift_logging.py

```diff
@@ -226,8 +220,10 @@ def create_instance_in_clusterlogging(sc_name=None):
             values, storage class and size details etc.
 
     """
+    num_of_worker_nodes = helpers.get_worker_nodes()
```

Comment:
num_of_worker_nodes is a 'list'. please consider changing this to `len(helpers.get_worker_nodes())`

Reply:
nodes_in_cluster = len(num_of_worker_nodes) + len(num_of_master_nodes) have this already in code

Reply:
agree with Karthick, please change it here to have `num_of_worker_nodes is a 'list'. please consider changing this to len(helpers.get_worker_nodes())` as it's more descriptive for the name of the variable. Than you can on line 225 directly use it without len as you will refer the num_of_worker_nodes directly.